app/styles/app/layouts/insights.scss

```diff
@@ -0,0 +1,213 @@
+
+.insights-controls {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .insights-dates {
+    font-size: 12px;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+
+  .insights-control-divider {
+    margin: 0 .5rem;
+    color: $dry-cement;
+  }
+
+  .insights-privacy-selector__selected {
+    font-weight: 700;
+  }
+
+  .ember-power-select-trigger {
+    display: inline-block;
+    vertical-align: middle;
+    border: none;
+    padding: 0 20px 0 0;
+    line-height: 1.45;
+
+    .ember-power-select-selected-item {
+      font-weight: 700;
+    }
+  }
+
+  .travistab-nav.travistab-nav--insights {
+    padding: 0;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0.5rem 0 1.5rem 0;
+
+    a {
+      color: $cement-grey;
+      cursor: pointer;
+    }
+
+    a:hover,
+    a:active,
+    li.active a {
+      color: $asphalt-grey;
+
+      &:after {
+        background-color: $asphalt-grey;
+        // height: 1px;
+        // bottom: -1px;
+      }
+    }
+  }
+}
+
+.insights-grid {
+  position: relative;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(4, minmax(50px, 1fr));
+
+  @media #{$medium-up} {
+    grid-gap: 40px 4%;
+  }
+
+  .insights-grid__overlay {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-color: rgba(255, 255, 255, 0.85);
+    text-align: center;
+
+    > h2 {
+      color: $oxide-blue;
+      font-size: 40px;
+      font-weight: 300;
+      margin: 55px 0 15px 0;
+    }
+
+    > p {

```

Comment:
I would suggest to avoid structural selectors when possible as they make styles fragile. Like wrap an element into another div and the styles will break.